.github/workflows/ci.yml

```diff
@@ -0,0 +1,41 @@
+name: CI
+on:
+  push:
+    branches:
+      - 'main'
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: 1.21.0
+      - name: go fmt
+        run: if [ "$(gofmt -s -l . | wc -l)" -gt 0 ]; then exit 1; fi
```

Comment:
Not a problem to solve here, but we should really have a standard `golang-ci` config file we use across all repos stored somewhere central.